lib/occ-helper.ts

```diff
@@ -38,7 +38,7 @@ export class OccHelper {
         edges.forEach((edge) => {
             const param1 = { current: 0 };
             const param2 = { current: 0 };
-            const crvHandle = this.occ.BRep_Tool.Curve_2(edge, param1 as any, param2 as any);
+            const crvHandle = this.occ.BRep_Tool.Curve_2(edge, param1.current, param2.current);
```

Comment:
I'm afraid this will not work, cause you pass numbers and not objects to the constructor. In C++ you can return values by passing references to variables as inputs. This is not really supported in javascript. Opencascade.js implemented a workaround to make this happen with those "current" properties.

Check these discussions on opencascade.js github:
https://github.com/donalffons/opencascade.js/issues/21
https://github.com/donalffons/opencascade.js/issues/55

You can also check this occt documentation page
https://dev.opencascade.org/doc/refman/html/class_b_rep___tool.html

You see that &First and &Last get values assigned in constructor itself instead of returning those normally as some single object. This is quite messed up and not "pure", but it's C++ and it can be done and OCCT team did that :)
Curve (const TopoDS_Edge &E, Standard_Real &First, Standard_Real &Last)

Reply:
Test showed me it didn't work! Thanks for the explanation. My suggestion is to leave a note with the discussion to github in the codebase so any future people can make sense of it.